airbyte-ci/connectors/pipelines/pipelines/models/ci_requirements.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import json
+from dataclasses import dataclass
+from importlib import metadata
+
+INFRA_SUPPORTED_DAGGER_VERSIONS = {
+    "0.6.4",
+    "0.9.5",
+}
+
+
+@dataclass
+class CIRequirements:
```

Comment:
I used a dataclass here in case we want to make the CI requirements payload a bit more complex. It does not have custom attributes at the moment but we can easily add some if needed.